graphql/schema.graphql

```diff
@@ -299,6 +300,11 @@ input IssueLinkInput {
   issueKey: String!
 }
 
+input SortOrder {
+  Key: String!
+  Direction: Int!
```

Comment:
This should be `TaskSortCategory` and `SortDirection` respectively since they are already defined enums.

Reply:
This will allow us to use the same generated constant types on the frontend.

Reply:
fixed